app/views/schools/dashboard/show.html.erb

```diff
@@ -1 +1,35 @@
-Dashboard goes here
```

Comment:
ups, I might have changed it to h1 to make the page pass automated accessibility check, sorry for conflict!